fission/function.go

```diff
@@ -432,3 +433,69 @@ func fnPods(c *cli.Context) error {
 
 	return err
 }
+
+func fnInvoke(c *cli.Context) error {
+	// Temporary, switch once internal function routes are extracted to separate internal router.
+	routerURL := os.Getenv("FISSION_ROUTER")
+	if len(routerURL) == 0 {
+		fatal("Need FISSION_ROUTER set to your fission router.")
+	}
+
+	fnName := c.String("name")
+	if len(fnName) == 0 {
+		fatal("Need name of function, use --name")
+	}
+
+	url := fmt.Sprintf("http://%s/fission-function/%s", routerURL, fnName)
+
+	resp := httpRequest(c.String("method"), url, c.String("body"), c.StringSlice("header"))
+
+	body, _ := ioutil.ReadAll(resp.Body)
+	fmt.Print(string(body))
+	defer resp.Body.Close()
+
+	return nil
+}
+
+func fnTest(c *cli.Context) error {
+	//client := getClient(c.GlobalString("server"))
+	fnName := c.String("name")
+
+	fnCreate(c)
```

Comment:
suggest removing

Reply:
I had thought that we wanted an end to end flow: create-test-delete but rechecked the description, will remove it.